bundles/org.openhab.binding.androidtv/README.md

```diff
@@ -30,37 +32,53 @@ This binding requires GoogleTV to be installed on the device (https://play.googl
-| appurl     | String | App URL                     |    N/A   |    RW    |
-| player     | Player | Player Control              |    RW    |    RW    |
-| power      | Switch | Power Control               |    RW    |    RW    |
-| volume     | Dimmer | Volume Control              |    RO    |    RO    |
-| mute       | Switch | Mute Control                |    RW    |    RW    |
+| Channel              | Type   | Description                          | GoogleTV | ShieldTV | PhilipsTV |
+|----------------------|--------|--------------------------------------|----------|----------|-----------|
+| keyboard             | String | Keyboard Data Entry                  |    RW    |    RW    |     RW    |
+| keypress             | String | Manual Key Press Entry               |    RW    |    RW    |     RW    |
+| keycode              | String | Direct KEYCODE Entry                 |    RW    |    RW    |     RW    |
+| pincode              | String | PIN Code Entry                       |    RW    |    RW    |     RW    |
+| app                  | String | App Control                          |    RO    |    RW    |     RW    |
+| appname              | String | App Name                             |    N/A   |    RW    |     RW    |
+| appurl               | String | App URL                              |    N/A   |    RO    |     N/A   |
+| appicon              | Image  | App Icon                             |    N/A   |    N/A   |     RO    |
+| player               | Player | Player Control                       |    RW    |    RW    |     RW    |
+| power                | Switch | Power Control                        |    RW    |    RW    |     RW    |
+| volume               | Dimmer | Volume Control                       |    RO    |    RO    |     RW    |
+| mute                 | Switch | Mute Control                         |    RW    |    RW    |     RW    |
+| tvChannel            | String | TV Channel Control                   |    N/A   |    N/A   |     RW    |
+| brightness           | Dimmer | Brightness Control                   |    N/A   |    N/A   |     RW    |
+| contrast             | Dimmer | Contrast Control                     |    N/A   |    N/A   |     RW    |
+| sharpness            | Dimmer | Sharpness Control                    |    N/A   |    N/A   |     RW    |
+| searchContent        | String | Google Assistant search              |    N/A   |    N/A   |     RW    |
+| ambilightPower       | Switch | Ambilight power control              |    N/A   |    N/A   |     RW    |
+| ambilightHuePower    | Switch | Ambilight + Hue power control        |    N/A   |    N/A   |     RW    |
+| ambilightStyle       | String | Ambilight Style plus algorithm used  |    N/A   |    N/A   |     RW    |
+| ambilightColor       | Color  | Color for all Ambilight Sides        |    N/A   |    N/A   |     RW    |
+| ambilightLeftColor   | Color  | Color for left Ambilight Side        |    N/A   |    N/A   |     RW    |
```

Comment:
See comment in thing-type.xml about naming conventions, when changed, these should match.